Hunting Queries/BehaviorAnalytics/Anomalous AAD Account Manipulation.yaml

```diff
@@ -31,7 +31,7 @@ query: |
  | extend Target =  iff(tostring(TargetResources.userPrincipalName) contains "#EXT#",replace("_","@",tostring(split(TargetResources.userPrincipalName, "#")[0])),TargetResources.userPrincipalName),tostring(TargetResources.userPrincipalName)
  | join kind=inner ( BehaviorAnalytics
  ) on $left._ItemId == $right.SourceRecordId
- | where UsersInsights.BlastRadius == "High" or ActivityInsights.FirstTimeUserPerformedAction == true
+ | where UsersInsights.BlastRadius == "High" or ActivityInsights contains "True"

```

Comment:
It is possible to use "has" here instead of "contains".  It is much more efficient.

Reply:
done